econml/bootstrap.py

```diff
@@ -35,16 +35,25 @@ class BootstrapEstimator(object):
 
     n_jobs: int, default: None
         The maximum number of concurrently running jobs, as in joblib.Parallel.
+
+    compute_means : bool, default: True
```

Comment:
Can we add some example of when this is useful?

Reply:
I'll add a comment, but the main use case for setting this to `False` is for our internal use of the estimator (where calling `effect_interval` on the boostrap wrapper would otherwise end up trying to take the mean of the `effect_interval` results from every wrapped instance, rather than computing the interval based on the `effect` results from every wrapped instance).